modules/hooks/init.js

```diff
@@ -604,14 +604,31 @@ export default function() {
       type: Boolean
     });
 
-    game.settings.register("wfrp4e", "useWoMOvercast", {
-      name: "SETTINGS.useWoMOvercast",
-      hint: "SETTINGS.useWoMOvercastHint",
-      scope: "world",
-      config: true,
-      default: false,
-      type: Boolean
-    });
```

Comment:
Was this a merge error? Feel like you need this

Reply:
Yes I need this, I think the auto merge with table settings didn't work well